sympy/integrals/tests/test_intpoly.py

```diff
@@ -179,6 +179,187 @@ def test_polytope_integrate():
+                      [5, 85, 78], [32, 81, 46], [10, 53, 48], [45, 78, 64],
+                      [7, 46, 66], [18, 48, 89], [37, 66, 85], [70, 89, 81],
+                      [29, 64, 53], [88, 74, 1], [38, 67, 48], [42, 83, 72],
+                      [57, 1, 85], [34, 48, 62], [59, 72, 87], [19, 62, 74],
+                      [63, 87, 67], [17, 85, 83], [52, 75, 1], [39, 87, 49],
+                      [22, 51, 40], [55, 1, 66], [29, 49, 64], [30, 40, 69],
+                      [13, 64, 75], [82, 69, 87], [7, 66, 51], [90, 85, 1],
+                      [59, 69, 72], [70, 81, 71], [88, 1, 84], [73, 72, 83],
+                      [54, 71, 68], [5, 83, 85], [50, 68, 69], [3, 84, 81],
+                      [57, 66, 1], [30, 68, 40], [28, 62, 48], [52, 1, 74],
+                      [23, 40, 51], [38, 48, 86], [9, 51, 66], [80, 86, 68],
+                      [11, 74, 62], [55, 84, 1], [54, 86, 71], [35, 64, 49],
+                      [90, 1, 75], [41, 71, 81], [39, 49, 67], [15, 81, 84],
+                      [61, 67, 86], [21, 75, 64], [24, 53, 43], [50, 69, 0],
+                      [37, 85, 47], [31, 43, 75], [61, 0, 67], [27, 47, 58],
+                      [10, 67, 53], [8, 58, 69], [90, 75, 85], [45, 91, 78],
+                      [80, 68, 0], [36, 66, 46], [65, 78, 85], [63, 0, 87],
+                      [32, 46, 56], [20, 87, 91], [14, 56, 68], [57, 85, 66],
+                      [33, 58, 47], [61, 86, 0], [60, 84, 77], [37, 47, 66],
+                      [82, 0, 69], [44, 77, 89], [16, 69, 58], [18, 89, 86],
+                      [55, 66, 84], [26, 56, 46], [63, 67, 0], [31, 74, 43],
+                      [36, 46, 84], [50, 0, 68], [25, 43, 53], [6, 68, 56],
+                      [12, 53, 67], [88, 84, 74], [76, 89, 77], [82, 87, 0],
+                      [65, 75, 78], [60, 77, 74], [80, 0, 86], [79, 78, 91],
+                      [2, 86, 89], [4, 91, 87], [52, 74, 75], [21, 64, 78],
+                      [18, 86, 48], [23, 58, 40], [5, 78, 83], [28, 48, 53],
+                      [6, 40, 68], [25, 53, 64], [54, 68, 86], [33, 83, 58],
+                      [17, 83, 47], [12, 67, 49], [41, 56, 71], [9, 47, 51],
+                      [35, 49, 91], [2, 71, 86], [79, 91, 83], [38, 86, 67],
+                      [26, 51, 56], [7, 51, 46], [4, 87, 72], [34, 89, 48],
+                      [15, 46, 81], [42, 72, 58], [10, 48, 67], [27, 58, 51],
+                      [39, 67, 87], [76, 81, 89], [3, 81, 77], [8, 69, 40],
+                      [29, 53, 49], [19, 77, 62], [22, 40, 56], [20, 49, 87],
+                      [32, 56, 81], [59, 87, 69], [24, 62, 53], [11, 62, 43],
+                      [14, 68, 71], [73, 91, 72], [13, 43, 64], [70, 71, 89],
+                      [16, 72, 69], [44, 89, 62], [30, 69, 68], [45, 64, 91]]
+    #  Actual volume is : 51.405764746872634
+    assert polytope_integrate(echidnahedron, 1) - 51.4057647468726 < 1e-15
```

Comment:
Use `abs(x-y)<eps`. Also, only 13 decimal digits are printed, so you must use something like 1e-12 or so. You can't do 1e-15, because that might fail sometimes.